src/main/java/by/radioegor146/NativeObfuscator.java

```diff
@@ -362,35 +362,29 @@ private static void processIndy(ClassNode classNode, String methodName, InvokeDy
         indyWrapper.instructions.add(new InsnNode(Opcodes.ARETURN));
         classNode.methods.add(indyWrapper);
     }
-    
-    private static final HashMap<Integer, String> INSTRUCTIONS = new HashMap<>();
-    
+
     /**
      * @param args the command line arguments
      * @throws java.io.IOException
      * @throws java.lang.IllegalAccessException
      */
     public static void main(String[] args) throws IOException, IllegalArgumentException, IllegalAccessException {        
-        for (Field f : Opcodes.class.getFields())
+        for (Field f : Opcodes.class.getDeclaredFields())
```

Comment:
```suggestion
        for (Field f : Opcodes.class.getDeclaredFields())
```
all fields in Opcodes are public so no need in `Class#getDeclaredFields()`